features/networking/ptp_operator.feature

```diff
@@ -11,3 +11,94 @@ Feature: PTP related scenarios
       | app=linuxptp-daemon |
     And status becomes :running of exactly 1 pods labeled:
       | name=ptp-operator |
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-25738
+  @admin
+  Scenario: Linuxptp run as slave can sync to master
+    Given I store the nodes in the clipboard
+    When I run oc create as admin over "<%= BushSlicer::HOME %>/testdata/networking/ptp_master.yaml" replacing paths:
+      | ["spec"]["recommend"][0]["match"][0]["nodeName"]      | <%= cb.nodes[0].name %> |
+    Then the step should succeed
+    When I run oc create as admin over "<%= BushSlicer::HOME %>/testdata/networking/ptp_slave.yaml" replacing paths:
+      | ["spec"]["recommend"][0]["match"][0]["nodeName"]      | <%= cb.nodes[1].name %> |
+    Then the step should succeed
+    Given I register clean-up steps:
+    """
+    When I run the :delete admin command with:
+      | object_type                 | PtpConfig          |
+      | object_name_or_id           | --all              |
+      | n                           | openshift-ptp      |
+    Then the step should succeed
+    """
+
+    And I wait up to 120 seconds for the steps to pass:
+    """
+    And I get the ptp logs of the "<%= cb.nodes[0].name %>"  node since "120s" ago
+    And the output should match:
+      | selected local clock (.*) as best master |
+    And I get the ptp logs of the "<%= cb.nodes[1].name %>"  node since "120s" ago
+    And the output should contain "CLOCK_REALTIME rms"
+    """
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-25740
+  @admin
+  Scenario: Linuxptp runs in transparent udp4 mode
+    Given I store the nodes in the clipboard
+    When I run oc create as admin over "<%= BushSlicer::HOME %>/testdata/networking/ptp_master.yaml" replacing paths:
+      | ["spec"]["recommend"][0]["match"][0]["nodeName"]      | <%= cb.nodes[0].name %> |
+      | ["spec"]["profile"][0]["ptp4lOpts"]                   | "-4"                    |
+    Then the step should succeed
+    When I run oc create as admin over "<%= BushSlicer::HOME %>/testdata/networking/ptp_slave.yaml" replacing paths:
+      | ["spec"]["recommend"][0]["match"][0]["nodeName"]      | <%= cb.nodes[1].name %> |
+      | ["spec"]["profile"][0]["ptp4lOpts"]                   | "-s -4"                 |
+    Then the step should succeed
+    Given I register clean-up steps:
+    """
+    When I run the :delete admin command with:
+      | object_type                 | PtpConfig          |
+      | object_name_or_id           | --all   |
```

Comment:
table `|` alignment issue and remove extra spaces

Reply:
Thanks, fixed it.